library/kani_macros/src/lib.rs

```diff
@@ -162,6 +162,26 @@ pub fn proof_for_contract(attr: TokenStream, item: TokenStream) -> TokenStream {
     attr_impl::proof_for_contract(attr, item)
 }
 
+/// `stub_verified(TARGET)` is a harness attribute (to be used on [`proof`] or
+/// [`proof_for_contract`] function) that replaces all occurrences of `TARGET`
+/// reachable from this harness with a stub generated from the contract on
+/// `TARGET`.
+///
+/// To create a contract for `TARGET` you must decorate it with at least one
```

Comment:
maybe say something more generic and add for example [`requires`] or [`ensures`]. I don't want us to update this comment when we add `assigns` or any other contract attribute.

Reply:
Sure. That works for me.